po/es.po

```diff
@@ -15260,39 +15262,39 @@ msgstr "No hay ninguna cámara con soporte de captura disponible para su uso..."
 
 #: ../src/views/view.c:2143
 msgid "Left click"
-msgstr "Clic izquierdo"
+msgstr "Clic"
 
 #: ../src/views/view.c:2146
 msgid "Right click"
-msgstr "Clic derecho"
+msgstr "Clic Secundario"
 
 #: ../src/views/view.c:2149
 msgid "Middle click"
-msgstr "Clic medio"
+msgstr "Clic Medio"
 
 #: ../src/views/view.c:2152
 msgid "Scroll"
 msgstr "Desplazamiento"
 
 #: ../src/views/view.c:2155
 msgid "Left double-click"
-msgstr "Doble clic izquierdo"
+msgstr "Pulsación Doble"
```

Comment:
I would translate it as "Doble clic" (https://es.wikipedia.org/wiki/Doble_clic).
It's uncommon to say "pulsación doble".

Reply:
As in other cases I don't just use a data source. It is also congruent with other related messages.